internal/cmd/serve/cmd.go

```diff
@@ -191,9 +191,7 @@ func watchFile(filename string, onChange func(in fsnotify.Event)) (*fsnotify.Wat
 
 func startServer(config *config.Config) error {
 	postProcess(config)
-	c := cors.New(cors.Options{
-		AllowedOrigins: []string{"*"},
```

Comment:
Missing defaults that were applied here

Reply:
The [`New`](https://github.com/rs/cors/blob/f9bce55a4e61e3d1a061993e3453eb9848fcdc4d/cors.go#L105) function of the rs/cors package handles all the default values for us, ie. even if no fields of `config.CorsConfig` are extracted from yaml, it will automatically assign the default values and return a cors handler.

Please have a look at this for reference: https://github.com/rs/cors/blob/f9bce55a4e61e3d1a061993e3453eb9848fcdc4d/cors.go#L122-L127

Reply:
The [New](https://github.com/rs/cors/blob/f9bce55a4e61e3d1a061993e3453eb9848fcdc4d/cors.go#L105) function of the rs/cors package handles all the default values for us, ie. even if no fields of `config.CorsConfig` are extracted from yaml, it will automatically assign the default values to them and return a cors handler.

Please have a look at this for reference: https://github.com/rs/cors/blob/f9bce55a4e61e3d1a061993e3453eb9848fcdc4d/cors.go#L122-L127

Reply:
The [New](https://github.com/rs/cors/blob/f9bce55a4e61e3d1a061993e3453eb9848fcdc4d/cors.go#L105) function of the rs/cors package handles all the default values for us.

So even if a field of `config.CorsConfig` has not been assigned any value in the yaml file, the `New` function will consider the default value for it (without the need of us explicitly assigning it) and then return the cors handler.

Please have a look at this for reference: https://github.com/rs/cors/blob/f9bce55a4e61e3d1a061993e3453eb9848fcdc4d/cors.go#L122-L127